### PR TITLE
Replace `make` as a script running with `just`

### PR DESCRIPTION
**Pull request by clux**
To follow conventions used in rest of org.

..triggered by needing to make a new target and stopped writing make pretty much everywhere else.

### CODE REVIEW

**Comment by clux** [comment]
Have added it to the devcontainer. The third party tools are currently documented on https://kube.rs/tools/ which sits under the contributing section. Do you think that's OK?

File: justfile [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    --k3s-arg "--no-deploy=traefik@server:*" \
+    --k3s-arg '--kubelet-arg=eviction-hard=imagefs.available<1%,nodefs.available<1%@agent:*' \
+    --k3s-arg '--kubelet-arg=eviction-minimum-reclaim=imagefs.available=1%,nodefs.available=1%@agent:*'
+
+bump-k8s:
+  #!/usr/bin/env bash
+  current=$(cargo tree --format "{f}" -i k8s-openapi | head -n 1)
+  next=${current::-2}$((${current:3} + 1))
+  fastmod -m -d . --extensions toml "$current" "$next"
+  fastmod -m README.md "$current" "$next"
```
Comment (kazk) [inline justfile:80]:
```suggestion
  fastmod -m README.md "$current" "$next"

# vim: set ft=make
```
This should make GitHub syntax highlight. See https://github.com/casey/just/blob/8ff7c87d962ffe8a22100592d506fde12f0e5156/justfile#L201

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -44,14 +44,18 @@ RUN scurl https://raw.githubusercontent.com/rancher/k3d/main/install.sh \
...
     | tar zvxf - --strip-components=1 -C $HOME/bin "cargo-deny-${CARGO_DENY_VERSION}-x86_64-unknown-linux-musl/cargo-deny"
 
 # Install cargo-tarpaulin
-ARG CARGO_TARPAULIN_VERSION=0.18.5
+ARG CARGO_TARPAULIN_VERSION=0.20.0
 RUN scurl "https://github.com/xd009642/tarpaulin/releases/download/${CARGO_TARPAULIN_VERSION}/cargo-tarpaulin-${CARGO_TARPAULIN_VERSION}-travis.tar.gz" \
     | tar xzvf - -C $HOME/bin
 
+ARG JUST_VERSION=1.1.3
+run scurl https://github.com/casey/just/releases/download/${JUST_VERSION}/just-${JUST_VERSION}-x86_64-unknown-linux-musl.tar.gz \
```
Comment (kazk) [inline .devcontainer/Dockerfile:57]:
```suggestion
RUN scurl https://github.com/casey/just/releases/download/${JUST_VERSION}/just-${JUST_VERSION}-x86_64-unknown-linux-musl.tar.gz \
```

**Review by kazk — APPROVED** [review]

File: justfile (hunk 1 above)
Comment (clux) [reply justfile:80]: Added the last 3 lines from that to make it pass. Unclear if it was just the extra `:` or the other two lines. But at least it works now.

**Review by kazk — APPROVED** [review]

**Pull request merged**